lib/confluence/vendor_specific.es6.js

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+require('./vendor_specific_data.es6.js');
+
+foam.CLASS({
+  package: 'org.chromium.apis.web',
+  name: 'VendorSpecific',
+  extends: 'org.chromium.apis.web.MetricComputer',
+  documentation: `VendorSpecific is a class that computes the Vendor Specific
+    API metrics from browserApiDAO. It shows the number of APIs
+    that only available in this browser.`,
```

Comment:
... actually, I think we need a more formal definition here. These APIs were shipped by a vendor at least 1 year ago, and haven't since been removed (i.e., appear in all of the last year's releases), and have not been visible in any other browser vendor during the last year's releases.